src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/TaskControllerTest.java

```diff
@@ -119,7 +129,7 @@ public void should_return_503_for_work_in_progress_endpoints() {
             .body("status", equalTo(HttpStatus.SERVICE_UNAVAILABLE.value()))
             .body("message", equalTo(responseMessage))
             .when()
-            .post("/task/{task-id}/unclaim", taskId);
+            .post("/task");
```

Comment:
Not too sure why this has changed to /task you have been implementing /task/{task-id}/unclaim so why change this to /task?